starters/express-apollo-prisma/README.md

```diff
@@ -85,10 +86,10 @@ git clone https://github.com/thisdot/starter.dev.git
 - Run `npm run start` to start the development server.
 - Open your browser to `http://localhost:4001` to see the API documentation with the existing endpoints.
 
-## Commands
+## Built-in Scripts
 
 - `npm run infrastructure:start` - Starts up a Mysql database and Redis instance for caching
```

Comment:
I just noticed all scripts are prefixed with `npm` or `npm run`. Shall we drop these, as it depends on which package manager one is using? For me, it would work just to have the script name.

If we decide against dropping, lines 100 through 102 and 105 need the `run` keyword added.

Reply:
@mbicknese I guess we can leave as is and put a note that someone should use their preferred package manager. Cause leaving with just `run` will be a bit confusing in the other sections e.g. seeding